src/work-allocation/components/work-allocation.components.module.ts

```diff
@@ -8,6 +8,7 @@ import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
 import * as fromComponents from '.';
 import * as fromPipes from '../pipes';
```

Comment:
The `fromPipes` references appear to have been removed from the `declarations` and `exports`... so is this still needed?

Reply:
For workallocation it is yes as this is how the pipes were originally gathered via the pipes folder. However, workallocation2 has additional functionality to just define a PipesModule (as it was created when pipes were needed to be gathered into one module for clone component). No changes are necessary as this is a different module to workAllocation2 and will be removed once workallocation2 is completed anyway. However, I can make the change to contain the workallocation1 pipes in a module quickly if you still want me to?

Reply:
No, this is cool. Thanks. 👍